lib/src/controller.dart

```diff
@@ -53,6 +59,7 @@ class MapViewController {
 
   void onMapViewerMessage(String type, Map<String, dynamic> payload) {
     MessageHandler(type).handleMessage(this, payload);
+    _internalMessageDelegate?.call(type, payload);
```

Comment:
Necesitamos pasarlle todos os mensaxes ? Non nos chegaria con enviarlle os do AR ?

Reply:
Ahora mismo si, pero a razón de ser deste método é precisamente non ter que facer máis cambios no plugin si o día de mañán o módulo de AR necesita outra mensaxe.